One of the test scripts requires Perl, and probably would work with any
version, but for dependency purposes, we need to pick one.

Not suitable for upstream.
diff -ru golang-1.5/test/errchk golang-1.5/test/errchk
--- golang-1.5/test/errchk	2015-09-08 18:24:03.000000000 -0700
+++ golang-1.5/test/errchk	2015-11-03 14:59:16.457207463 -0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/perl5/5.20/bin/perl
 # Copyright 2009 The Go Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
